website/docs/r/mks_nodegroup_v1.html.markdown

```diff
@@ -58,6 +58,8 @@ resource "selectel_mks_nodegroup_v1" "nodegroup_1" {
 
 * `nodes_count` - (Required) Number of worker nodes in the node group. Changing this resizes the node group if `enable_autoscale` is false.
 
+* `install_nvidia_device_plugin` - (Required) Boolean flag indicates if nvidia device plugin and GPU drivers installation was requested. Positive value is only suitable for `flavor_id` with GPU. In other cases with positive value - 400 (Bad request) status code will be returned from API.
```

Comment:
* `install_nvidia_device_plugin` - (Required) Enables or disables installation of the NVIDIA Device Plugin and GPU drivers.
Boolean flag:
-`true` — for flavors with GPU enables installation of the NVIDIA Device Plugin and GPU drivers.
- `false` — for flavors without GPU and flavors with GPU disables installation of the NVIDIA Device Plugin and GPU drivers. Learn more about [manual installation of GPU drivers](https://docs.selectel.ru/en/cloud/managed-kubernetes/node-groups/install-gpu-drivers/).